client/src/models/enums/contactQuestioningExcelFields.ts

```diff
@@ -25,19 +24,28 @@ export const ContactedPersonFieldMapper: ContactedPersonExcel= {
     doesHaveBackgroundDiseases: 'האם סובל ממחלות רקע',
     isolationAddress: {
         city: 'יישוב השהייה בבידוד',
```

Comment:
Aren’t the other address fields part of the isolation address as well?

Reply:
in this context the isolationAddress's only use is on the validateContact function that needs to accept a city field in this format and show the appropriate message.